extensions/podman/packages/extension/src/podman-install.ts

```diff
@@ -807,3 +816,61 @@ class WSLVersionCheck extends BaseCheck {
     });
   }
 }
+
+export class HyperVCheck extends WindowsCheck {
+  title = 'Hyper-V installed';
+
+  async execute(): Promise<extensionApi.CheckResult> {
+    if (!(await this.isUserAdmin())) {
+      return this.createFailureResult({
+        description: 'You must have administrative rights to run Hyper-V Podman machines',
+        docLinksDescription: 'Contact your Administrator to setup Hyper-V.',
+        docLinks: {
+          url: 'https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v',
+          title: 'Hyper-V Manual Installation Steps',
+        },
+      });
+    }
+    if (!(await this.isHyperVinstalled())) {
+      return this.createFailureResult({
+        description: 'Hyper-V is not installed on your system.',
+        docLinksDescription: 'call DISM /Online /Enable-Feature /All /FeatureName:Microsoft-Hyper-V in a terminal',
+        docLinks: {
+          url: 'https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v',
+          title: 'Hyper-V Manual Installation Steps',
+        },
+      });
+    }
+    if (!(await this.isHyperVRunning())) {
+      return this.createFailureResult({
+        description: 'Hyper-V is not running on your system.',
+        docLinksDescription: 'call sc start vmms in a terminal',
+        docLinks: {
+          url: 'https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v',
+          title: 'Hyper-V Manual Installation Steps',
+        },
+      });
+    }
+    return this.createSuccessfulResult();
+  }
+
+  private async isHyperVinstalled(): Promise<boolean> {
+    try {
+      const result = await extensionApi.process.exec('powershell.exe', [
+        '@(Get-Wmiobject Win32_ComputerSystem).HypervisorPresent',
```

Comment:
I have uninstalled hyperv (from the Turn Windows features on or off), rebooted, but this command still returns True

Reply:
Good catch, updated